tests/foreman/cli/test_contentview.py

```diff
@@ -4156,7 +4156,7 @@ def test_positive_remove_cv_version_from_multi_env_capsule_scenario(self):
                     }
                 )
             # Synchronize the capsule content
-            Capsule.content_synchronize({'id': capsule['id'], 'organization-id': org['id']})
+            # Capsule.content_synchronize({'id': capsule['id'], 'organization-id': org['id']})
```

Comment:
Why not base whether this function is called on the BZ state associated with the failure that's projected to be fixed in a future version?

Reply:
Also, was there additional conversation about the suggestions from @JacobCallahan and @mirekdlugosz to actually update the `content_synchronize` method to base its actions on if a sync is already in progress?

Reply:
Conversation was already discussed with @JacobCallahan about this.  I will look into it and work with what was suggested.  I will update this PR once the pytest conversion PR is merged.

Reply:
I think I just merged that pr